stripe/src/main/java/com/stripe/android/model/PaymentMethod.java

```diff
@@ -1146,6 +1160,97 @@ public Fpx build() {
         }
     }
 
+    public static final class SepaDebit extends PaymentMethodTypeImpl {
+        private static final String FIELD_BANK_CODE = "bank_code";
+        private static final String FIELD_BRANCH_CODE = "branch_code";
+        private static final String FIELD_COUNTRY = "country";
+        private static final String FIELD_FINGERPRINT = "fingerprint";
+        private static final String FIELD_LAST4 = "last4";
+
+        @NonNull public final String bankCode;
+        @NonNull public final String branchCode;
+        @NonNull public final String country;
+        @NonNull public final String fingerprint;
+        @NonNull public final String last4;
+
+        SepaDebit(
+                @Nullable String bankCode,
+                @Nullable String branchCode,
+                @Nullable String country,
+                @Nullable String fingerprint,
+                @Nullable String last4
+        ) {
+            super(Type.SepaDebit);
+            this.bankCode = bankCode != null ? bankCode : "";
+            this.branchCode = branchCode != null ? branchCode : "";
+            this.country = country != null ? country : "";
+            this.fingerprint = fingerprint != null ? fingerprint : "";
+            this.last4 = last4 != null ? last4 : "";
```

Comment:
what's the reasoning behind defaulting all these parameters to empty string? are those all optional?
if so, wouldn't it better to use null?